doc/rules/no_empty_block.md

```diff
@@ -0,0 +1,38 @@
+# No empty block
+
+## Rule id
+no-empty-block
+
+## Description
+Disallows empty blocks. Blocks with a todo comment inside are not considered empty.
```

Comment:
How about treating blocks with any comment as non-empty? Majority of empty blocks are empty catch blocks which are intentionally empty so there is no sense in adding todo comment there but "intentionally do nothing"-comments seem to fit fine. Or there should be an option to not report empty catch blocks, just like in TSLint.